homework-classes/RepoView.js

```diff
@@ -19,8 +19,54 @@
      * @param {Object} repo A repository object.
      */
     render(repo) {
-      // TODO: replace this comment and the console.log with your own code
-      console.log('RepoView', repo);
+      function convertDateTime(dateTimeTxt) {
```

Comment:
When working with classes, class methods are preferred over embedded functions, such as this `convertDateTime()` function. This would work like this:

```js
render(repo) {
  ...
  createAndAppend('span', updatedTime, {
    text: this.convertDateTime(repo.updated_at),
    class: 'texts',
  });
}

convertDateTime(dateTimeTxt) {
  const dateTime = new Date(dateTimeTxt);
  return dateTime.toLocaleString();
}
```

Or better still, since the `convertDateTime` method does not require access to the `this` value in the method bodyl you could make this method **static**.

```js
render(repo) {
  ...
  createAndAppend('span', updatedTime, {
    text: RepoView.convertDateTime(repo.updated_at),
    class: 'texts',
  });
}

static convertDateTime(dateTimeTxt) {
  const dateTime = new Date(dateTimeTxt);
  return dateTime.toLocaleString();
}
```

Reply:
dear @remarcmij thank you for the valuable feedbacks during the whole JS3. I will improve the codes based on your feedback